corehq/apps/events/models.py

```diff
@@ -50,36 +58,18 @@ class Event:
     case: CommCareCase
     manager: WebUser
     is_open: bool = True
-
     attendee_list_status: AttendeeListStatus = 'Not started'
 
-    # Todo: implement the following
-    # attendance_takers: set[CommCareUser]
-    # possible_attendees: set[Attendee]
-
-    @classmethod
-    def get_obj_from_data(cls, data) -> Event:
-        event_obj = cls()
-
-        event_obj.domain = data['domain']
-        event_obj.name = data['name']
-        event_obj.start_date = data['start_date']
-        event_obj.end_date = data['end_date']
-        event_obj.attendance_target = data['attendance_target']
-        event_obj.sameday_reg = data['sameday_reg']
-        event_obj.track_each_day = data['track_each_day']
-        event_obj.manager = data['manager']
-        event_obj.is_open = data.get('is_open', Event.is_open)
-        event_obj.attendee_list_status = data.get('attendee_list_status', Event.attendee_list_status)
-
-        if 'case' in data:
-            event_obj.case = data['case']
+    expected_attendees: set[Attendee] = []
+    attendance_takers: set[CommCareUser] = []
```

Comment:
Don't set default values to mutable data types. [Bad things](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) can happen.
```suggestion
    expected_attendees: set[Attendee] | None = None
    attendance_takers: set[CommCareUser] | None = None
```